apps/wallet/src/background/keyring/DerivedAccount.ts

```diff
@@ -25,7 +25,7 @@ export class DerivedAccount implements Account {
         derivationPath: string;
         keypair: Keypair;
     }) {
-        this.type = AccountType.IMPORTED;
```

Comment:
oops 😨, I'm kind of surprised this didn't cause any issues... thanks for fixing this. It might be good to initialize the account type on line 17 for all accounts like we're doing for Qredo so a mistake like this doesn't happen again

Reply:
I don't think it's used, we mainly use the serialised value that was correct